### PR TITLE
Added redirect to products details

### LINKED ISSUE #392: Add redirect to the single product page in user order

### PR DESCRIPTION
**Pull request by D1nviz**
https://github.com/user-attachments/assets/02f783a9-71ce-48dd-8751-156c9e88b043

### CODE REVIEW

**Review by mhevyk — APPROVED** [review]

**Review by heshvab — APPROVED** [review]

File: code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx [hunk 1]
```diff
@@ -49,14 +56,14 @@ const OrderProductItem = ({
           component="p"
           fontWeight="extra-bold"
         >
-          {formatPrice(product.price)} x {quantity}
+          {formatPrice(productPrice)} x {quantity}
```
Comment (dmtrth25) [inline code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx]: create some variable with logic here and render instead double render like `{} x {}`
It'll be more readable

File: code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx [hunk 2]
```diff
@@ -17,14 +19,19 @@ const OrderProductItem = ({
   quantity,
   price
 }: OrderProductItemProps) => {
+  const { price: productPrice, id, image, name, description } = product;
```
Comment (dmtrth25) [inline code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx]: if you give some renaming for price `price: productPrice`
maybe also give it for other properties line `name`, `image`, `description`
Or leave just `price` instead `price: productPrice` just everything in one approach

File: code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx (hunk 1 above)
Comment (D1nviz) [reply code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx]: Thank you. Moved

File: code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx (hunk 2 above)
Comment (D1nviz) [reply code/src/containers/order-item/components/order-product-item/OrderProductItem.tsx]: Changed naming

**Review by dmtrth25 — APPROVED** [review]

**Pull request merged**